contents/tutorials/compare-aws-bedrock-prompts.md

```diff
@@ -0,0 +1,331 @@
+---
+title: How to compare AWS Bedrock prompts
+date: 2024-08-29
+author:
+  - lior-neu-ner
+tags:
+  - product analytics
+  - AI engineering
+---
+
+Evaluating LLM prompts is crucial for determining whether your modifications are genuinely enhancing your application. In this tutorial, we explore three methods to assess prompts by comparing their LLM outputs. Namely:
```

Comment:
"modifications are genuinely enhancing your application." is a bit wordy
```suggestion
Evaluating LLM prompts is crucial for determining whether they are useful for users. In this tutorial, we explore three methods to assess prompts by comparing their LLM outputs. Namely:
```

Reply:
Lol I don't remember writing this at all. I think cursor included this and I hit tab without realizing  lol